omero/users/cli/installation.rst

```diff
@@ -4,16 +4,49 @@ Installation
 .. note:: The |CLI| is currently untested on Windows
     but may be supported in the future.
 
-We recommend installing in a Python virtual environment.
-Since OMERO 5.6, only Python 3 is supported::
+Since OMERO 5.6, only Python 3 is supported.
+We assume that you have already installed Python 3.6.
 
-    python3.6 -mvenv venv
-    . venv/bin/activate
+We recommend installing omero-py and the |CLI| plugins
+in a Python virtual environment.
+You can either create one using ``venv`` or ``conda`` (preferred).
```

Comment:
``You can either create one using...`` -> ``You can create one using either....``